api/factory/ginwebserver/ginWebServerHandler.go

```diff
@@ -0,0 +1,233 @@
+package ginwebserver
```

Comment:
added unit test for `httpServer` component. For the rest of them, I don't see a way of reaching this because we rely on `gin`'s component and cannot extract interfaces for custom types everywhere